[RHDHBUGS-758]: Add Operator procedure for air-gapped dynamic plugin install (#2260)
redhat-developer/red-hat-developers-documentation-rhdh: modules/shared/proc-install-dynamic-plugins-in-an-air-gapped-environment-by-using-the-operator.adoc

:_mod-docs-content-type: PROCEDURE

[id="install-dynamic-plugins-in-an-air-gapped-environment-by-using-the-operator_{context}"]
= Install dynamic plugins in an air-gapped environment by using the Operator

[role="_abstract"]

> In an air-gapped environment, you can install external dynamic plugins by configuring a custom NPM registry for the {product-short} Operator. You create a Kubernetes secret containing a `.npmrc` file and mount it into the `install-dynamic-plugins` init container.

Reviewer: ```suggestion
To install external dynamic plugins in an air-gapped environment, configure a custom NPM registry by mounting a secret into the {product-short} Operator.
```
